File: IDTO-master/IDTO MDT/IdtoWsComms/src/org/battelle/idto/ws/otp/data/ServerInfo.java

package org.battelle.idto.ws.otp.data;

public class ServerInfo{
   	private String cpuName;
   	private Number nCores;
   	private ServerVersion serverVersion;

 	public String getCpuName(){
		return this.cpuName;
	}
	public void setCpuName(String cpuName){
		this.cpuName = cpuName;
	}
 	public Number getNCores(){
		return this.nCores;
	}
	public void setNCores(Number nCores){
		this.nCores = nCores;
	}
 	public ServerVersion getServerVersion(){
		return this.serverVersion;
	}
	public void setServerVersion(ServerVersion serverVersion){
		this.serverVersion = serverVersion;
	}
}

File: IDTO-master/IDTO Azure Hosted Systems/IDTO.RouteAggregationLibrary/OpenTripPlanner/Model/ServerInfo.cs
namespace IDTO.RouteAggregationLibrary.OpenTripPlanner.Model
{
    public class ServerInfo
    {
        public ServerVersion serverVersion { get; set; }
        public string cpuName { get; set; }
        public int nCores { get; set; }

        public ServerInfo()
        {
            serverVersion = new ServerVersion();
        }
    }
}